metaflow/datastore/flow_datastore.py

```diff
@@ -158,7 +159,7 @@ def get_latest_task_datastores(
                     elif fname == TaskDataStore.METADATA_DATA_SUFFIX:
                         # This somewhat breaks the abstraction since we are using
                         # load_bytes directly instead of load_metadata
-                        with open(path, "rb") as f:
+                        with open(path, encoding="utf-8") as f:
```

Comment:
I am not familiar with this part of the code. @romain-intel is this the right thing to do here?

Reply:
This looks ok yes. The data is stored with `save_metadata` which does a json.dumps followed by an encode("utf-8") so this should be reasonable to open as text with that encoding.